gradle.properties

```diff
@@ -8,6 +8,6 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
 springBootVersion=2.1.1.RELEASE
 
 eventuateUtilVersion=0.1.0.RELEASE
-eventuateMessagingPartitionManagementVersion=0.2.0.RELEASE
+eventuateMessagingPartitionManagementVersion=0.1.0-SNAPSHOT
```

Comment:
There is no PR for eventuate-common-messaging.
Also, eventuateMessagingPartitionManagementVersion should be renamed to eventuateCommonMessagingVersion

Reply:
There was not PR, because it required released common libraries.
Could you please release it?
I will rename.